pkg/kubectl/resource/helper.go

```diff
@@ -82,15 +92,26 @@ func (m *Helper) List(namespace, apiVersion string, selector labels.Selector, ex
 	return req.Do().Get()
 }
 
-func (m *Helper) Watch(namespace, resourceVersion, apiVersion string, labelSelector labels.Selector) (watch.Interface, error) {
+func (m *Helper) Watch(namespace, resourceVersion, apiVersion string, labelSelector labels.Selector, fieldSelector string) (watch.Interface, error) {
```

Comment:
Because of the other PR converting labels.Selector to string, this should probably be merged after that one (which is getting close).